resources/qml/MessageView.qml

```diff
@@ -50,7 +50,12 @@ ListView {
 
     Shortcut {
         sequence: StandardKey.Cancel
-        onActivated: chat.model.reply = undefined
+        onActivated: {
+            if (chat.model.edit)
```

Comment:
If someone is editing a reply (is that possible?) will this cause some kind of issue where only one gets set to undefined?  wouldn't it be easier to just set both to undefined?

Reply:
That's intentional. When you are editing a message, you can change what you were replying to. This includes deleting the reply. In which case it makes sense to do that with Esc, before you cancel the edit with a second Esc, imo.